example_configs/podman-quadlets/lldap-db.container

```diff
@@ -0,0 +1,38 @@
+###
+### lldap-db - podman quadlet file
+### https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html
+###
+
+[Unit]
+Description=LLDAP-DB container
+Documentation=https://hub.docker.com/_/postgres
+# likely only needed for Podman < 5.3 (see containers/podman#22197 and #24637)
+After=podman-user-wait-network-online.service
+
+[Service]
+Restart=on-failure
+
+[Container]
+AutoUpdate=registry
+ContainerName=lldap-db
+Image=docker.io/postgres:17
+
+# this is optional to keep db/web services separate
+Network=backend.network
+
+# health check
+HealthCmd=pg_isready -U lldapuser -d lldap
+HealthInterval=5s
+HealthRetries=3
+HealthStartPeriod=15s
+HealthTimeout=30s
+
+# environmental variables
```

Comment:
Environment.

Environmental is for nature, trees and such :)

Reply:
thank you! sorry about that!